### PR TITLE
Add details about ACME EAB and HMAC related configuration.  Also add details about Google CA.

### PR DESCRIPTION
**Pull request by banaag**

### CODE REVIEW

File: cmd/webpkgserver/README.md [hunk 1]
```diff
@@ -132,6 +133,84 @@ information needed for creating signed exchanges:
...
+[SXG.ACME]
+  # Enable webpkgserver to attempt to auto renew certificates using ACME.
+  #Enable = false 
+
+  # The path to the Certificate Signing Request PEM file.
+  # Required when ACME is enabled.
+  CSRFile = "path/to/csr.pem"
+
+  # The ACME discovery URL. It is specified by the Certificate Authority that
+  # doles out your certificates. As of August 2020, only DigiCert supports
```
Comment (twifkak) [inline cmd/webpkgserver/README.md:151]: fix "only DigiCert" here

File: cmd/webpkgserver/README.md [hunk 2]
```diff
@@ -52,6 +52,7 @@ in the repository root.
 [golang]: https://golang.org/doc/install
 [CanSignHttpExchanges]: https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#cross-origin-cert-req
 [DigiCert]: https://www.digicert.com/account/ietf/http-signed-exchange.php
+[Google]: https://cloud.google.com/blog/products/identity-security/automate-public-certificate-lifecycle-management-via--acme-client-api
```
Comment (twifkak) [inline cmd/webpkgserver/README.md:55]: I recall there being a slightly different process for SXG certs, whereas it seems that blog post only documents TLS certs.

I think it's a different ACME discovery URL? Whatever it is, can you document it in this README?

File: cmd/webpkgserver/README.md (hunk 1 above)
Comment (banaag) [reply cmd/webpkgserver/README.md]: Done.

File: cmd/webpkgserver/README.md (hunk 2 above)
Comment (banaag) [reply cmd/webpkgserver/README.md:56]: Added.

**Review by twifkak — APPROVED** [review]
Sorry for the delay!

**Pull request merged**